conans/test/integration/command/new_test.py

```diff
@@ -1,265 +1,120 @@
 import os
 import textwrap
-import unittest
 
 import pytest
-from parameterized import parameterized
 
 from conans import __version__ as client_version
 from conans.test.utils.test_files import temp_folder
 from conans.test.utils.tools import TestClient
 from conans.tools import save
-from conans.util.files import load
 
 
-class NewCommandTest(unittest.TestCase):
+class TestNewCommand:
+    def test_new_cmake_lib(self):
+        client = TestClient()
+        client.run("new cmake_lib --name=pkg --version=1.3")
+        conanfile = client.load("conanfile.py")
+        assert "CMakeToolchain" in conanfile
```

Comment:
Perhaps, it could be safer to check the whole line `tc = CMakeToolchain(self)`

Reply:
This is just a very quick test to check the file is not empty. There are full tests building the result, this is not an issue.